img/d/displate.com.svg

```diff
@@ -0,0 +1 @@
23-.07.3-.15.58-.22.8-.45 1.53-.9 2.77-1.21 3.14.38.51 1.96 1.16 2.79 1.16.45 0 1.05-1.96 1.59-4.37.07-.44.22-.95.3-1.39 8.3-2.7 9.05-2.9 11.54-3.79.08.22.08.44.08.66.07.22.07.51.07.73a8.35 8.35 0 0 1-3.25 6.7 10.3 10.3 0 0 1-1.58 1.1c-4.6 2.77-11.39 3.21-13.2 1.03-.08-.16-.3-.6-.38-.89 0-.36-1.28.15-.9 1.46 0 .07 0 .07.07.15.23.88.6 1.67.99 2.19 1.58 2.04 5.65 2.18 9.88 1.1 1.13-.3 10.41-3.22 11.61-10.66a10.39 10.39 0 0 0-.14-3.79v-.06c-.08-.23-.08-.52-.24-.81 1.74-.58 2.65-.73 2.5.15l-1.06 5.9s-.98 4.6 2.94 4.6c1.44 0 2.64-.66 3.47-1.47 1.96 2.34 6.41 1.53 7.92-.8a3.34 3.34 0 0 0 .38-.73c.3-.65.23-1.24.07-1.74zm-26.1-3.35s-2.79.95-4.52 1.46c.38-2.04.52-4.01.45-4.89-.07-.43-.53-.87-1.13-1.09 0 0-2.04-.87-1.89.08l.08.22c.22 1.53-.15 3.93-.6 6.12-1.66 0-4.38-.37-4.31-2.7 0-3.42 4-7.36 10.71-7a8.88 8.88 0 0 1 7.78 4.96c.07.07.07.15.15.29 0 .07.07.15.07.22a297.15 297.15 0 0 0-6.79 2.33Zm56.88 6.78c-.37-.22-.6-.51-.6-.51-2.33-2.11-1.2-6.2-.9-7.07.3-.58.15-1.24-.3-1.3h-1.13c-.45-1.76-1.66-2.93-3.17-2.93-2.5 0-5.06 2.92-5.74 6.56a8.63 8.63 0 0 0-.08 2.77c-.82 1.17-1.65 1.9-2.33 1.9-.9 0-1.06-1.61-.83-3.72l1.06-5.69 1.05-5.83S58.7 0 57.5 0h-2.12l-1.35 7.87-.39 2.04a7.28 7.28 0 0 1-1.5 2.4l.15-.87c.38-2.76-.6-5.17-2.57-5.68-1.13-.3-2.4.15-3.47 1.16v-.07s.3-1.3-.9-1.3h-2.11L40.59 20.7c0 .14-.08.21-.08.36-.07.36-.07.65-.07.87-.08 1.32 2.63 2.04 3.02 1.76-.08-.52.07-1.32.22-2.34v-.15l.75-4.15a3.36 3.36 0 0 0 1.82 1.38c2.1.52 4.45-1.46 5.43-4.45a5.28 5.28 0 0 0 1.43-1.16c-.07.95-.15 1.82-.07 2.26a3.73 3.73 0 0 0 3.92 3.35c1.13 0 2.18-.87 3.02-2.03.6 1.16 1.58 1.96 2.86 1.96 1.36 0 2.8-.95 3.92-2.4 1.14 2.7 2.87 2.62 4 2.1h.08l.15-.06c-.15 0 .83-.59.37-.88zm-25.42-.94c-.75-.22-1.13-1.54-.9-3.22l.08.08a4.24 4.24 0 0 0 1.58 1.24c.52.14.98.22 1.43.29-.75 1.17-1.51 1.83-2.19 1.6zm3.03-3.8a6.21 6.21 0 0 1-.39 1.1 4.28 4.28 0 0 1-.9-.3 4.44 4.44 0 0 1-1.43-.94 3.7 3.7 0 0 1-.53-.58l-.23-.37c.68-2.26 1.96-3.94 2.94-3.64.9.21 1.21 2.33.53 4.73zm17.5-.3c-.53 2.42-1.74 4.24-2.8 4.17-1.06-.07-1.44-2.12-.98-4.45.53-2.4 1.74-4.23 2.8-4.15 1.05.07 1.5 2.03.97 4.44zM89.02 10.35c-.76 2.19-2.94 6.27-5.36 6.27-1.28 0-1.96-1.02-2.1-2.18.97.06 2.1-.52 3.01-1.39a6.1 6.1 0 0 0 1.5-2.04l.23-.44c.83-1.97.6-3.8-.38-4.66a1.35 1.35 0 0 0-.52-.37h-.08c-.15-.08-.37-.15-.53-.22h-.07a3.73 3.73 0 0 0-3.47 1.1c-1.59 1.3-2.79 4.37-2.79 7.07v.58c-.9 1.45-1.96 2.48-2.72 2.48-.9 0-1.05-1.61-.83-3.72l1.06-5.69.07-.36h1.29c.75 0 1.43.07 1.5.14.3-.65.16-1.3-.3-1.38h-2.26l.76-4.23s.3-1.31-.9-1.31H74l-.98 5.54h-2.95c-.45 0-.75-.08-.9-.15-.15.15-.38 1.24.68 1.4h3.02l-.23 1.08-.9 5.03c-.08 1.02-.16 1.83-.08 2.34a3.73 3.73 0 0 0 3.92 3.35c1.28 0 2.5-1.17 3.47-2.55a4.06 4.06 0 0 0 3.85 2.47c5.13 0 7.09-7.5 7.09-7.5-.15-.3-.53-.66-.98-.66zm-7.32.87c.08-.43.23-.87.45-1.38.68-1.46 1.74-2.55 2.42-2.33.68.22.68 1.53 0 3.06a6.55 6.55 0 0 1-1.29 1.9c-.6.65-1.73.65-1.73.65-.15-.87.15-1.9.15-1.9z"/><path d="m27 1.53 1.6-.15.67-1.38.68 1.38 1.58.23-1.13 1.01.23 1.53-1.36-.72-1.43.72.3-1.53Z" fill="#1185ed"/></svg>
```

Comment:
How about this one, as shown in their favicon:
![](https://raw.githubusercontent.com/phallobst/phallobst/main/resterampe/displate.com.svg)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="498.2" height="498.2" viewBox="0 0 373.6 373.6"><path d="M186 2.2c-.1.2-5.2.6-11.3 1-6 .3-12 .7-13 1l-6.6 1a225.7 225.7 0 0 0-35 8.2l-9 3a281.1 281.1 0 0 0-45 23.4C33.5 61.6 13 86.3 4.2 114.4L1.8 124 .6 130c-.8 4.7-.7 18.3 0 24.7C4 180.1 21 197.4 50 204.7c10.3 2.6 19.9 3.5 36.5 3.4 12.3 0 13.7.1 13.3 1.6l-3.1 11.5A310 310 0 0 1 81 267.4l-2 4.7c0 .3 2.6 2.2 5.8 4.3 11.9 8 32.3 14.3 35.9 11.3a49 49 0 0 0 7.8-15c5.9-18 9.4-31.9 14.5-57.7 1.7-8.5 3.3-15.7 3.6-16.2.6-1 52.5-18.4 53-17.8 7.2-6 9.8-10.4-4.4-8.7l-16.3 5.3a205.2 205.2 0 0 1-30.4 9c-.2-.3 0-2.7.5-5.4s1.3-7.6 1.6-10.9l1-8c.2-1 .8-4.5 1-7.5l1-8A194 194 0 0 0 155 107c-.5-1.7-2.4-4.7-4.2-6.5a57.9 57.9 0 0 0-19.1-10.2l-4.5-1.2c-.8-.3-4-.7-7.1-1-7.4-.7-8.5.6-7.1 8.4 1 5.6.9 36.4-.2 41l-1.1 8a357.9 357.9 0 0 1-8.5 47.7c-.5.5-3.3.8-6.2.7-21.4-.4-41.6-7-51.3-16.8-7.7-7.6-10.7-15.7-10.6-28 .3-29 18.7-59.8 48.9-81.3a178.4 178.4 0 0 1 50.8-23.5l11-2.4c13.4-3 20.7-3.6 43.4-3.5 18.2 0 22.8.4 34.5 3l4 .8a133.9 133.9 0 0 1 86.8 70.1 100.1 100.1 0 0 1 12.3 52.5c-1 17.7-2.7 27.5-7.6 42.1-13.3 39.8-47.7 73-98 94.1-18 7.7-41.4 15-55.5 17.3l-5 1-6 1-7.5 1.1c-15.3 2.4-41 2.3-56-.2a119.2 119.2 0 0 1-23-6.9 56.6 56.6 0 0 1-15.4-10c-3-3.2-8.6-13.8-8.6-16.7 0-1.2-.6-1.5-2.2-1-5.4 1.2-9.8 8-9.8 15 0 6 4.7 20.8 10 31.7 10.2 20.7 25.5 31 54.5 36.4 8.3 1.6 13.6 2 30.5 1.6 11.3-.2 22-.6 24-1l18-3c30-4.9 71.5-21.3 105.5-41.8a279.8 279.8 0 0 0 59.9-50 165.4 165.4 0 0 0 38-81.7c1.3-8 1.4-41.7.3-46.5-.3-1-.8-4.5-1.2-7.5-3-22.6-12.4-43.7-29.6-66A173.5 173.5 0 0 0 257 13.3c-5.7-1.8-11.4-3.6-12.8-3.8L237.7 8c-4-1-12.2-2.6-19.5-3.7-2-.3-9.3-.8-16.5-1a196.2 196.2 0 0 1-15.7-1z"/><path d="m200.3 178.5-3 18.4c-1.3 8.6-2.6 15.8-2.4 16.2.3.4 7.4-2.8 15.9-7.3 15.2-7.9 18-8.9 19.9-7 1 .8 8.7 5.1 21.9 12l7.8 4-.6-3.3-1-5.3-1-6-2.2-12.8c-1.6-9.4-1.4-10 3.4-14.3 2.9-2.5 21.2-20.5 21.2-20.8 0-.2-2.1-.7-4.7-1l-20.1-3-11.4-1.8-8-16.3c-4.3-9-8.1-16.4-8.4-16.4s-4 7.1-8.2 15.7c-4.2 8.7-8.1 16.1-8.5 16.4-.5.4-3 1-5.6 1.3l-7.1 1c-7.3 1.3-21.4 3-22.2 2.8-2.3-.6.2 2.5 9.3 11.4l10 10z" fill="#009fe3"/></svg>
```